### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by dobooth**
## Purpose of this pull request

- #5649
- #5652
- #5615
- #5648
- #5614
- #5597
- #5616
- #5523
- #5592
- #5588
- #5589
- #5590
- #5591
- #5595
- #5596
- #5598
- #5583

### CODE REVIEW

File: guides/v2.2/javascript-dev-guide/javascript/custom_js.md [hunk 1]
```diff
@@ -142,10 +142,14 @@ For information about how to initialize your custom JS component in a `.phtml` t
 
 ## Disable default Magento JS {#disable_default_js}
 
-To disable the auto-loading of default Magento JS components and widget
-initialization:
+To disable the auto-loading of default Magento JS components and widget initialization:
+
+1. Create a `requirejs-config.js` file with the following content:
+
+```javascript
```
Comment (shrielenee) [inline guides/v2.2/javascript-dev-guide/javascript/custom_js.md]: Does this need to be indented to appear correctly?

File: guides/v2.3/security/two-factor-authentication.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ After enabling and configuring 2FA for your Magento instance, Magento Admin user
 | : --------- | : --------- | : ------- |
 | [Google Authenticator](https://support.google.com/accounts/answer/1066447?co=GENIE.Platform%3DAndroid&hl=en)  | Generate and enter code from mobile app Requirements: Enable in Admin  | `google`|
 | [Authy](https://authy.com/)  | SMS, call, token, and one touch <br/>Requirements: Enable in Admin and API keys  | `authy` |
-| [U2F Keys](https://docs.magento.com/m2/2.1/ee/user_guide/stores/security-two-factor-authentication-use.html)   | Physical device to authenticate, like [YubiKey](https://www.yubico.com/). <br/>Requirements: Enable in Admin  | `u2fkey` |
+| [U2F Keys](https://docs.magento.com/m2/ee/user_guide/stores/security-two-factor-authentication-use.html?#u2f-key)   | Physical device to authenticate, like [YubiKey](https://www.yubico.com/). <br/>Requirements: Enable in Admin  | `u2fkey` |
```
Comment (shrielenee) [inline guides/v2.3/security/two-factor-authentication.md]: I would remove the  ?

URL still works without it.

File: guides/v2.2/javascript-dev-guide/javascript/custom_js.md (hunk 1 above)
Comment (dobooth) [reply guides/v2.2/javascript-dev-guide/javascript/custom_js.md]: Done

File: guides/v2.3/security/two-factor-authentication.md (hunk 2 above)
Comment (dobooth) [reply guides/v2.3/security/two-factor-authentication.md]: Nailed it

**Comment by dobooth** [comment]
running tests

**Comment by dobooth** [comment]
running tests

**Pull request merged**